source/reference/minio-server/minio-server.rst

```diff
@@ -160,16 +125,15 @@ The command accepts the following arguments:
 
          minio server --address "minioip:9000" ~/miniodirectory
 
-   If omitted, :mc:`minio <minio server>` binds to port ``9000`` on all
-   configured IP addresses or hostnames on the host machine.
+   If omitted, :mc:`minio <minio server>` binds to port ``9000`` on all configured IP addresses or hostnames on the host machine.
 
 .. mc-cmd:: --console-address
    :optional:
 
    Specifies a static port for the embedded MinIO Console.
 
-   Omit to direct MinIO to generate a dynamic port at server startup. The
-   MinIO server outputs the port to the system log.
+   Omit to direct MinIO to generate a dynamic port at server startup. 
+   The MinIO server outputs the port to the system log.
```

Comment:
This is about line 148 and others that I'm not able to comment on. (This is in original text, not the new changes.)

`_(Optional)_` is rendering with literal underscores, should switch it to `:optional:` if that works in those locations.

Reply:
Ah, nice catch. Replacing with surrounding `*` instead.